stdpopsim/catalog/AraTha/demographic_models.py

```diff
@@ -102,6 +102,8 @@ def _sma_1pop():
         )
     ]
 
+    mutation_rate = 7.1e-9
```

Comment:
Maybe we should separate out these catalog changes from the code updates? These should be QC'd more closely and would require looking up the original papers etc, which is a lot to do at once.

Specifying a mutation rate shouldn't be necessary for the models, it can default to None so the basic change PR can go through.

Reply:
Done - I'll open a separate PR with these rates and their tests for equality.